docs/developer-guide/release.md

```diff
@@ -25,7 +25,7 @@ Replacing:
 ```
 - [ ] Create an issue with this checklist in the release milestone.
 - [ ] If major release (YYYY.XX.00), create a branch (**YYYY.XX.xx**)  (`xx` is really `xx`, example: 2018.01.xx)
-- [ ]  If major release,Change [QA Jenkins job](http://build.geo-solutions.it/jenkins/view/MapStore2/job/MapStore2-QA-Build/) to build the new branch, enable the job continuous deploy
+- [ ] If major release,Change [QA Jenkins job](http://build.geo-solutions.it/jenkins/view/MapStore2/job/MapStore2-QA-Build/) to build the new branch, enable the job continuous deploy and update the `TAG_NAME` parameter in the [build job for the docker image](https://build.geo-solutions.it/jenkins/view/MapStore2/job/MapStore2%20QA%20Docker%20Image/configure)
```

Comment:
Can you indicate how to update build job for the docker image TAG_NAME? which format? It is the current or the next version?
 - v2019.01.xx
 - v2019.01.00
 - 2019.01.xx
 - 2019.01.00
(what it builds?)